lib/browser/helpers.js

```diff
@@ -205,7 +206,11 @@ module.exports = {
   checkIfVisible: function (selector) {
     return this.execute(function (selector) {
       var element = document.querySelector(selector);
-      return !!element && window.getComputedStyle(element).display !== 'none';
+      if(!!element){
+        return element.offsetWidth > 0 && element.offsetHeight > 0;
```

Comment:
@looshi Interesting, I didn't know this trick. So you're saying that this one also takes ancestor visibility into account?